electrode-ota-server-ccm/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "electrode-ota-server-ccm",
+  "version": "1.0.0",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "ota-mocha",
+    "build": "ota-babel",
+    "coverage": "ota-nyc",
+    "prepublish": "npm run build"
+  },
+  "dependencies": {
+    "@walmart/electrode-ccm-store": "^1.0.0",
```

Comment:
can't access this

Reply:
but OTA-Public-instace does include private fileservice package, ain't it?

Reply:
they put it there and not here.

Reply:
moved it to our public instance